modules/blog/index.php

```diff
@@ -145,8 +145,8 @@
     foreach ($entries as $entry) {
         $blogbody = '';
         if ($entry['image']!='') {
```

Comment:
Just use !empty() here.

But overall the idea is incorrect. We already have fields processing in method _processRow or something like that. You MUST NOT process fields several times, this should be done in methods that get your data and process multilingual fields, etc.